docs/template_plugin/src/template_preprocessing.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "template_preprocessing.hpp"
+
+#include <ngraph/pass/manager.hpp>
+#include <ngraph/opsets/opset3.hpp>
+
+#include "transformations/preprocessing/mean_image_or_value.hpp"
+#include "transformations/preprocessing/std_scale.hpp"
+
+using namespace ngraph;
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::AddPreprocessing, "AddPreprocessing", 0);
```

Comment:
Can this pass be moved to transformations library as well?

Reply:
This pass works on IE structures, we can move it only to plugin API. Is it OK?

Reply:
OK. At least it will be shared with other plugins.

Reply:
Moved to plugin api.